rg_instructor_analytics/templates/rg_instructor_analytics/cohort.html

```diff
@@ -11,14 +11,14 @@
     from openedx.core.lib.url_utils import quote_slashes
     from openedx.core.lib.xblock_utils import wrap_xblock
 %>
-<h1>${_("Cohort")}</h1>
-<h2 class="enrollment-title">${_("Lorem ipsum dolor sit amet")}</h2>
-<span class="enrollment-title-text">${_("Lorem ipsum, dolor sit amet consectetur adipisicing elit. Adipisci ad perspiciatis culpa molestiae dolorem omnis quis maxime consequuntur. Ipsam, ratione.")}</span>
+<h1>${_("Clusters")}</h1>
+<h2 class="enrollment-title">${_("Keep an eye on your students’ progress")}</h2>
+<span class="enrollment-title-text">${_("This report constantly divides your students’ progress in clusters: from low-performing to high-performing groups. You can take action right away - send appropriate email to clusters in couple of clicks.")}</span>
```

Comment:
Please make the text be not longer than 120 chars.

Reply:
All done